main.js

```diff
@@ -76,11 +79,25 @@ function displayPalette(event) {
   for (i = 0; i < currentPalette.colors.length; i++) {
     paletteContainer.innerHTML +=
     `<article class="color-container" data-id="${currentPalette.colors[i].id}">
-      <div class="color-box color1" style="background:${currentPalette.colors[i].hexcode};"></div>
+      <div class="color-box" style="background:${currentPalette.colors[i].hexcode};"></div>
       <span>
         <p class="hexcode">${currentPalette.colors[i].hexcode}</p>
         <i class="lock">🔓</i>
       </span>
     </article>`;
   }
 }
+
+function savePalette() {
+  savedPalettes.push(currentPalette);
+  currentPalette = new Palette();
+    savedPaletteContainer.innerHTML +=
+    `<article class="saved-palette" data-id="${currentPalette.colors[0].id}">
+       <div class="saved-color-box" style="background:${currentPalette.colors[0].hexcode};"></div>
+       <div class="saved-color-box" style="background:${currentPalette.colors[1].hexcode};"></div>
```

Comment:
I'd love to see us refactor this so that you make a function like displayPalettes(){} that does the html part, but re-renders all the palettes in savedPalettes each time it's called. Then when you call savePalette(), you would just do 

1. push current palette into saved palettes
2. reset the current palette to a new instance of Palette
3. invoke the displayPalettes function, which updates that part of the screen based on the new contents of savedPalettes